frontend/src/common/enums/app/data-status.enum.ts

```diff
@@ -0,0 +1,8 @@
+enum DataStatus {
```

Comment:
Merge with development branch and remove similar things
We already have it in the dev branch